doc/manual/rl-next/pretty-json.md

```diff
@@ -0,0 +1,10 @@
+---
+synopsis: "Prettified JSON output on the terminal"
+issues: 12555
+prs: [12652]
+---
+
+Scripts are mostly unaffected because for those, stdout will be a file or a pipe, not a terminal, and for those, the old single-line behavior applies.
```

Comment:
```suggestion
This makes the output easier to read.

Scripts are mostly unaffected because for those, stdout will be a file or a pipe, not a terminal, and for those, the old single-line behavior applies.
```